Resources/Prototypes/DeltaV/Body/Organs/vulpkanin.yml

```
@@ -13,3 +27,9 @@
         reagents:
         - ReagentId: UncookedAnimalProteins
           Quantity: 5
+  - type: Metabolizer
+    maxReagents: 3
+    metabolizerTypes: [ Animal ]
```

yamllint

[error] too many spaces inside brackets

(brackets)